Integrations/FeedOffice365/FeedOffice365.yml

```diff
@@ -64,39 +63,33 @@ configuration:
   - F - Reliability cannot be judged
   required: true
   type: 15
-- display: ""
-  name: expiration
-  defaultvalue: ""
-  type: 17
+- name: expiration
   required: false
-- display: Bypass exclusion list
+  type: 17
+- additionalInfo: When selected, the exclusion list will be ignored for this feed.
+  display: Bypass exclusion list
   name: bypassExclusionList
-  defaultvalue: ""
-  type: 8
   required: false
-  additionalinfo: When selected, the exclusion list will be ignored for this feed.
-- display: Mark as reputation
+  type: 8
+- defaultvalue: feedInstanceReputationNotSet
+  display: Mark as reputation
```

Comment:
So was this. Should be `Indicator reputation`. Also the `additionalinfo` field for this parameter should be `Indicators from this integration instance will be marked with this reputation`. See [here](https://demisto.slack.com/archives/CRT0G4Y82/p1579437079007200) for more details.